package.json

```diff
@@ -15,9 +15,7 @@
   "watch": {
     "push-build": {
       "patterns": [
-        "src",
-        "plugins",
-        "service-worker"
```

Comment:
I am guessing with all the SW changes you tested changing the SW during development?

Reply:
plugins and service-worker directories have been moved into src, so we only need to watch src now.